defaults/main.yml

```diff
@@ -243,11 +243,11 @@ docker_ferment_wrapper: '{{ (ansible_local.root.lib
 #   Configuration of other Ansible roles
 # ----------------------------------------
 
-# .. envvar:: docker_etc_services_dependent_list
+# .. envvar:: docker__etc_services_dependent_list
 #
 # Configuration for ``debops.etc_services`` role which registers port numbers
 # for Docker REST API.
-docker_etc_services_dependent_list:
+docker__etc_services_dependent_list:
```

Comment:
This variable should be named `docker__etc_services__dependent_list` since the list in the corresponding role is named `etc_services__dependent_list` and you want it passed on through the playbook.